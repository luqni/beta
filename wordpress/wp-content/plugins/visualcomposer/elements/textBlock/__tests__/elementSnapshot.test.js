/* global expect, it */
import React from 'react'
import './__mocks__/api'

import TextBlockElement from '../textBlock/component'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const text = 'Hello jest <b>from</b> react'

  const tree = renderer
    .create(<TextBlockElement atts={{ output: text }} id={'123456789'} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
