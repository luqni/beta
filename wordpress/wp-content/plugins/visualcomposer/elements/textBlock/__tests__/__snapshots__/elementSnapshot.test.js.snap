// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="vce-text-block"
>
  <div
    className="vce-text-block-wrapper vce"
    data-vce-do-apply="all el-123456789"
    id="el-123456789"
  >
    Hello jest &lt;b&gt;from&lt;/b&gt; react
  </div>
</div>
`;
